contracts/src/kleros-v1/KlerosV1Governor.sol

```diff
@@ -0,0 +1,190 @@
+    /** @dev Constructor. Before this contract is made the new governor of KlerosLiquid, the evidence period of all subcourts has to be set to uint(-1).
+     *  @param _klerosLiquid The trusted arbitrator to resolve potential disputes.
+     *  @param _governor The trusted governor of the contract.
+     *  @param _foreignGateway The trusted gateway that acts as an arbitrator, relaying disputes to v2.
+     */
+    constructor(
+        IKlerosLiquid _klerosLiquid,
+        address _governor,
+        IArbitrator _foreignGateway
+    ) {
+        klerosLiquid = _klerosLiquid;
+        governor = _governor;
+        foreignGateway = _foreignGateway;
+    }
+
+    /** @dev Lets the governor call anything on behalf of the contract.
+     *  @param _destination The destination of the call.
+     *  @param _amount The value sent with the call.
+     *  @param _data The data sent with the call.
+     */
+    function executeGovernorProposal(
+        address _destination,
+        uint256 _amount,
+        bytes calldata _data
+    ) external onlyByGovernor {
+        (bool success, ) = _destination.call{value: _amount}(_data); // solium-disable-line security/no-call-value
+        require(success, "Call execution failed.");
+    }
+
+    /** @dev Changes the `governor` storage variable.
+     *  @param _governor The new value for the `governor` storage variable.
+     */
+    function changeGovernor(address _governor) external onlyByGovernor {
+        governor = _governor;
+    }
+
+    /** @dev Relays disputes from KlerosLiquid to Kleros v2. Only disputes in the evidence period of the initial round can be realyed.
+     *  @param _disputeID The ID of the dispute as defined in KlerosLiquid.
+     */
+    function relayDispute(uint256 _disputeID) external {
+        require(klerosLiquidDisputeIDtoGatewayDisputeID[_disputeID] == 0, "Dispute already relayed");
+        IKlerosLiquid.Dispute memory KlerosLiquidDispute = klerosLiquid.disputes(_disputeID);
+        (uint256[] memory votesLengths, , uint256[] memory totalFeesForJurors, , , ) = klerosLiquid.getDispute(
+            _disputeID
+        );
+
+        // Check that no juror was yet drawn. Add a function to finalize the juror drawing and move the dispute to vote.
+        require(KlerosLiquidDispute.period == IKlerosLiquid.Period.evidence, "Invalid dispute period.");
+        require(votesLengths.length == 1, "Cannot relay appeals.");
+
+        klerosLiquid.executeGovernorProposal(address(this), totalFeesForJurors[0], "");
+
+        uint256 minJurors = votesLengths[0];
+        bytes memory extraData = abi.encode(KlerosLiquidDispute.subcourtID, minJurors);
+        uint256 arbitrationCost = foreignGateway.arbitrationCost(extraData);
+        require(totalFeesForJurors[0] >= arbitrationCost, "Fees not high enough."); // If this doesn't hold at some point, it could be a big issue.
+        uint256 gatewayDisputeID = foreignGateway.createDispute(KlerosLiquidDispute.numberOfChoices, extraData);
```

Comment:
This will revert unless the arbitrationCost is payed

Reply:
Good catch. Should be solved now.